### PR TITLE
new sourcew: Kirklees council 

### PR DESCRIPTION
**Pull request by SplinterHead**

### CODE REVIEW

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/kirklees_gov_uk.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+from datetime import datetime
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+
+_LOGGER = logging.getLogger(__name__)
+
+TITLE = "Kirklees Council"
+DESCRIPTION = "Source for waste collections for Kirklees Council"
+URL = "https://www.kirklees.gov.uk"
+TEST_CASES = {
+    "Test_001": {"door_number": 12, "postcode": "200003806457"},
```
Comment (5ila5) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/kirklees_gov_uk.py:14]: `door_number` must be `door_num` (must match `__init__` parameters)

**Comment by 5ila5** [comment]
Please also add to info.md the best way to update md files is just calling the [update_docu_links.py script](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/update_docu_links.py). ([from contribution guide](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/contributing_source.md#update-links-in-readmemd-and-infomd))
Even if I change your test variable names I still get an error. Have you tested your source?
[How to test](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/contributing_source.md#test-the-new-source-file)

**Comment by SplinterHead** [comment]
Apologies, I updated the testcases after committing the code. The updated tests should now be available

**Comment by 5ila5** [comment]
Thanks for Your Contribution.

Removed your Logger and added an additional test case

**Pull request merged**